subprojects/build-cache-base/build-cache-base.gradle.kts

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018 the original author or authors.
+ * Copyright 2017 the original author or authors.
```

Comment:
Why this change? Shouldn't that be 2020?

Reply:
I have no idea how we ended up with this number. Updated to `2020`.